modules/Microsoft.DocumentDB/databaseAccounts/.test/plain/deploy.test.bicep

```diff
@@ -0,0 +1,85 @@
+targetScope = 'subscription'
```

Comment:
This PR is about converting what already existing to the new self-contained dependencies approach, so probably this comment is a bit out of scope. Nevertheless, after a very high level review, I'm noticing the following:

We here have 4 tests:
- 1 for a plain resource, where only the parent resource type is deployed together with diagnostic settings, locks and rbac
- 3 additional tests, testing each a different child resource: mongodb, gremlindb and sqldb

This seems a bit inconsistent with the majority of the other modules. The most consistent approach I'd see here would be to have:

- a min test: similar to plain, but removing diagnostics, locks and rbac, hence testing only required parameters
- a common test: deploying gremlindb, sqldb and mongodb child resources together with additional features like locks, diagnostics and rbac

What's your take on that? If you agree we can either rework tests in this PR or open a new feature request to align afterwards.

Reply:
A few thoughts:
- I'd see this also as out-of-scope but something we can address in a separate issue
- For that issue we should have a chat with Margarita/Marius as they may have been a rationale here
- Can one CosmosDB have multiple types deployed at once? Never used the service so I'm dependent on others knowing